arrow/util/protobuf_reflect_test.go

```diff
@@ -97,10 +100,11 @@ func AllTheTypesFixture() Fixture {
 		Any:      anyMsg,
 		//Breaks the test as the Golang maps have a non-deterministic order
 		//SimpleMap:   map[int32]string{99: "Hello", 100: "World", 98: "How", 101: "Are", 1: "You"},
-		SimpleMap:   map[int32]string{99: "Hello"},
-		ComplexMap:  map[string]*util_message.ExampleMessage{"complex": &exampleMsg},
-		SimpleList:  []string{"Hello", "World"},
-		ComplexList: []*util_message.ExampleMessage{&exampleMsg},
+		SimpleMap:     map[int32]string{99: "Hello"},
+		ComplexMap:    map[string]*util_message.ExampleMessage{"complex": &exampleMsg},
+		SimpleList:    []string{"Hello", "World"},
+		ComplexList:   []*util_message.ExampleMessage{&exampleMsg},
+		JsonlikeField: s,
```

Comment:
doesn't this also need to get added to the expected arrow schema side of things?

Reply:
yep, I am just making the recursion error to start whilst I think about it 👍